CodeEdit/Features/Settings/Pages/SourceControlSettings/SourceControlGitView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  SourceControlGitView.swift
+//  CodeEdit
+//
+//  Created by Raymond Vleeshouwer on 02/04/23.
+//
+
+import SwiftUI
+
+struct SourceControlGitView: View {
+    @AppSettings var settings
+
+    @State
+    var ignoredFileSelection: IgnoredFiles.ID?
+
+    var body: some View {
+        SettingsForm {
+            Section {
+                gitAuthorName
+                gitEmail
+            }
+            Section {
+                preferToRebaseWhenPulling
+                showMergeCommitsInPerFileLog
+            }
+        }
+    }
+}
+
+private extension SourceControlGitView {
+    private var gitAuthorName: some View {
+        TextField("Author Name", text: $settings.sourceControl.git.authorName)
+    }
+
+    private var gitEmail: some View {
+        TextField("Author Email", text: $settings.sourceControl.git.authorEmail)
```

Comment:
Add an onSubmit modifier here so when the return key is pressed the text view loses focus like so:
```swift
.onSubmit {
    NSApplication.shared.sendAction(#selector(NSApplication.resignFirstResponder), to: nil, from: nil)
}
```

Reply:
Should this be done on every TextField in Settings? This would seem redundant. Should we create a modifier for this? `.loadFocusOnSubmit()`?

Reply:
@thecoolwinter I am not quite sure that worked...

https://user-images.githubusercontent.com/806104/233466866-2cbe26e8-f53f-4411-ae3f-fd1e49f6bf61.mov